core/backend/src/CloudStorageBackend.ts

```diff
@@ -179,7 +188,7 @@ export class CloudStorageTileUploader {
     this._activeUploads.delete(containerKey + resourceKey);
   }
 
-  public cacheTile(tokenProps: IModelRpcProps, treeId: string, contentId: string, content: Uint8Array, guid: string | undefined) {
+  public cacheTile(tokenProps: IModelRpcProps, treeId: string, contentId: string, content: Uint8Array, guid?: string, metadata?: object) {

```

Comment:
Technically this is a quasi-breaking change to the method signature here...

For example, if you define:
```ts
function foo(a: string, b?: string) { /* ... */ }
function bar(a: string, b: string | undefined) { /* ... */ }
```
The types of foo and bar are similar, but there's a key distinction between _optional_ (`b?: string`) and _potentially undefined_ (`b: string | undefined`):
```ts
foo("a", "b");  // OK
bar("a", "b");  // OK
foo("a", undefined);  // OK
bar("a", undefined);  // OK
foo("a");  // OK
bar("a");  // TypeScript Error: Expected 2 arguments, but got 1
```

I think `guid` was non-optional because at this point you should pretty much always supply one, and if you pass undefined that should be a very explicit choice...

It's perfectly fine for `metadata` to be optional though - we actually need it to be to avoid making this a breaking change.